Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
```

Comment:
It would be beneficial to check if any commands could include error handling or output control to make debugging easier during development.